include/wx/generic/infobar.h

```diff
@@ -75,6 +75,21 @@ class WXDLLIMPEXP_CORE wxInfoBarGeneric : public wxInfoBarBase
     // get the currently used effect animation duration
     int GetEffectDuration() const { return m_effectDuration; }
 
+    // Whether the "Do not show this again" checkbox was checked at the
+    // time of the window being closed.
+    // This should be called in a client's handler for the
+    // wxID_CLOSE button being clicked.
+    bool IsDontShowAgainChecked() const { return m_dontShowAgain; }
+
+    // Whether the "Do not show this again" checkbox is being shown.
+    bool IsIncludingDontShowAgainCheckbox() const { return m_includeDontShowAgain; }
+
+    // Sets whether the "Do not show this again" checkbox should be shown.
+    void IncludeDontShowAgainCheckbox(const bool includeCheckbox)
```

Comment:
Top-level const is ignored and shouldn't be used.
```suggestion
    void IncludeDontShowAgainCheckbox(bool includeCheckbox)
```